include/engine/routing_algorithms/alternative_path.hpp

```diff
@@ -239,6 +240,8 @@ class AlternativeRouting final
         std::vector<NodeID> preselected_node_list;
         for (const NodeID node : via_node_candidate_list)
         {
+            if( node == middle_node )
```

Comment:
Run clang-format with correct style file, this uses two spaces and no enclosing `{}`.